spicy/runtime/include/sink.h

```diff
@@ -62,8 +62,8 @@ auto _connectUnit(UnitRef<U>& unit) {
     auto self = hilti::rt::ValueReference<U>::self(&*unit);
 
     auto& state = unit->__sink;
-    state = new sink::detail::State();                   // NOLINT
-    state->resumable = (*parse2)(self, state->data, {}); // Kick-off parsing with empty data.
+    state = new sink::detail::State();                       // NOLINT
+    state->resumable = (*parse2)(self, state->data, {}, {}); // Kick-off parsing with empty data.
```

Comment:
Should this pass on `unit`'s context?

Reply:
I'm not quite sure yet what to do about sink's and contexts (if anything), but just passing on the context isn't unlikely to work because the types would probably not match.